### ISSUE TITLE
Connecting to the Shared/Local vault not working on mobile

### ISSUE THREAD

**Issue by ravinderjangra** [maidsafe/safe_app_csharp#138]

**Comment by ravinderjangra** [comment]
PR on SCL to resolve this: https://github.com/maidsafe/safe_client_libs/pull/1051

**Comment by ravinderjangra** [comment]
Once that is merged then another PR to use the latest SCL in `safe-api`: https://github.com/maidsafe/safe-cli/pull/293
